src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api.Types/Responses/GetApprenticeshipsResponse.cs

```diff
@@ -7,5 +7,7 @@ public class GetApprenticeshipsResponse
         public IEnumerable<ApprenticeshipDetailsResponse> Apprenticeships { get; set; }
         public int TotalApprenticeshipsFound { get; set; }
         public int TotalApprenticeshipsWithAlertsFound { get; set; }
+        public int TotalApprenticeships { get; set; }
+        public int TotalApprenticeshipsWithAlerts { get; set; }
```

Comment:
Are these variables necessary? As far as I can see they're not used anywhere

Reply:
Provider commitments uses the top one and I've now removed the alerts one since we found out it was no longer needed. This value has now been wired up as well as part of MF-404